packages/core/src/index.ts

```diff
@@ -549,7 +525,7 @@ export class Hopr extends EventEmitter {
     }
   }
 
-  private async strategyOpenChannel(status: OutgoingChannelStatus) {
+  /*private async strategyOpenChannel(status: OutgoingChannelStatus) {
```

Comment:
?

Reply:
This is kept commented-out intentionally as a reference for converting the inner logic in Rust.